src/pages/hotel/booking/rate-details/RateDetails.scss

```diff
@@ -0,0 +1,172 @@
+.main-content-book {
+  border-style: solid;
+  background-color: white;
+  display: flex;
+  flex-direction: row;
+  padding: 80px;
+  margin-bottom: 10px;
+  margin-top: -30px;
+}
+.side-content {
+  padding: 2px;
+  float: left;
+  margin-top: 20px;
+}
+.booking-details {
+  width: 350px;
+  border: 1px solid rgb(220 220 220);
+  padding: 4px;
+}
+.booking-content {
+  margin-top: 10px;
+  margin-right: 10px;
+  margin-left: 10px;
+}
+.heading {
+  font-size: 1.25rem;
+  color: rgb(0 0 0);
+  font-weight: 700;
+  white-space: nowrap;
+  margin-top: 2px;
+}
+.dates {
+  display: flex;
+  flex-direction: row;
+  padding: 3px;
+  justify-content: space-between;
+  margin-top: 2px;
+  margin-bottom: 7px;
+  width: 100%;
+}
+.check-in-date {
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+  white-space: nowrap;
+  // margin-right: 15px;
+  font-size: 1.1rem;
+}
+.check-out-date {
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+  white-space: nowrap;
+  font-size: 1.1rem;
+}
+.date-title {
+  font-weight: 700;
+  font-size: 1.2rem;
+}
+.book-date {
+  font-weight: 700;
+}
+.border-mid {
+  background: rgb(220 220 220);
+  padding: 0.2px;
+}
+.stay-length {
+  color: rgb(0 0 0);
+  font-size: 1rem;
+  font-weight: 500;
+  margin-top: 10px;
+  font-size: 1.25rem;
+}
+.nights-count {
+  font-weight: 700;
+  font-size: 1.1rem;
+}
+.divider {
+  background: rgb(220 220 220);
+  padding: 0.2px;
+}
+.mid-layer {
+  margin-bottom: 20px;
+}
+.bottom-layer {
+  margin-top: 20px;
+}
+.selection {
+  font-weight: 700;
+  font-size: 1.1rem;
+}
+.rate {
+  margin-top: 10px;
+  //margin-left: 65px;
+  border: 1px solid rgb(220 220 220);
+  padding: 10px;
+}
+.rate-content {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-right: 10px;
+  margin-left: 10px;
+}
+.rate-heading {
+  font-size: 1.25rem;
+  color: rgb(0 0 0);
+  font-weight: 700;
+  white-space: nowrap;
+  margin-bottom: 10px;
+  margin-top: 4px;
+}
+.room-price {
+  display: flex;
+  flex-direction: row;
+}
+.room {
+  margin-right: 80px;
+}
+.room-heading {
+  margin-right: 90px;
+  font-size: 1.25rem;

```

Comment:
common styles can be grouped up instead of repeating it in all styles. 
ex: font-size, color and font-weight is similar as rate heading. 
so you can completely remove these from room-heading and also add rate heading as className with room-heading element.
ex: `<h2 className="room-heading rate-heading">Price</h2>`
**Also You can check these with all other styles and style sheets as well**

Reply:
Done